lib/transaction.js

```diff
@@ -122,6 +125,33 @@ module.exports = function transaction() {
   });
 };
 
+transaction.start = function start(modelBase) {
+  if (!modelBase) {
+    return Promise.reject(new Error('objection.transaction.start: provide at least one Model class to bind to the transaction'))
+  }
+
+  if (!utils.isSubclassOf(modelBase, Model)) {
+    return Promise.reject(new Error('objection.transaction.start: all but the last argument should be Model subclasses'));
+  }
+
+  var knex = modelBase.knex();
+
+  return new Promise(function(resolve, reject) {
+    knex.transaction(function (trx) {
+        modelBase = modelBase.knex(trx);
```

Comment:
Yes I know, I did it like this for my test, and I have wait your review, I think is possible to do similar think like the ````.transaction```` method.